imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc

```diff
@@ -0,0 +1,138 @@
+---
+title: Partner Conducted KYC
+slug: guides/kyc-modes/partner-conducted-kyc
+---
+
+KYC (Know Your Customer) verification process is necessary to ensure the
+security and legitimacy of card usage. It is an essential prerequisite for using
+the Immersve card.
+
+Below, we've outlined the steps for completion of this verification
+process, assuming all necessary details, such as document scans and biometrics,
+have been collected and verified.
+
+{% note %}
+Partner account must be configured to allow partner conducted KYC checks. This is
+subject to an agreement between Immersve and the partner. Ask Immersve support
+for more details.
+{% /note %}
+
+See the [KYC API endpoint references](https://docs.immersve.com/api-reference/kyc/).
+
+- Set up additional environment variables
+
+{% code title="Environment variable setup" %}
+    ```bash
```

Comment:
Do we wan this indent? We need a convention.

Reply:
I think it fits the code block better